pep-0394.txt

```diff
@@ -22,8 +22,9 @@ Python interpreter (i.e. the version invoked by the ``python`` command).
 
 * ``python2`` will refer to some version of Python 2.x.
 * ``python3`` will refer to some version of Python 3.x.
-* for the time being, all distributions *should* ensure that ``python``
-  refers to the same target as ``python2``.
+* for the time being, all distributions *should* ensure that ``python``,
+  if installed, refers to the same target as ``python2``, unless the system
+  administrator or user deliberately override this.
```

Comment:
So this should probably change to ensuring that `python`, *if it exists*, runs Python 2 unless (a) the user explicitly overrides it, or (b) a venv is active.